fictadvisor-web/src/types/user.ts

```diff
@@ -44,3 +44,12 @@ export interface UserRemainingSelective {
 export interface UserAdmin extends Omit<User, 'firstName' | 'lastName'> {
   state: UserGroupState;
 }
+
+export interface SimplifiedUser {
```

Comment:
if it is possible you should use extends User instead of simple declaration of interface